sdk/python/kfp/v2/components/experimental/component_spec.py

```diff
@@ -56,51 +64,69 @@ class InputValuePlaceholder(BasePlaceholder):
     """Class that holds input value for conditional cases.
 
     Attributes:
-        input_value: name of the input.
+        input_name: name of the input.
     """
-    input_value: str
+    input_name: str = pydantic.Field(alias='inputValue')
+
+    class Config:
+        allow_population_by_field_name = True
```

Comment:
nit: I think you could add the config in the base class `BasePlaceholder` so that you don't need to repeat it in the sub-classes. If you need this for all classes, consider create a base model: https://pydantic-docs.helpmanual.io/usage/model_config/#change-behaviour-globally

Reply:
Thanks! Used the global base class.